### DISCUSSION TITLE
help with react native lesson 6.4

### DISCUSSION THREAD

**Discussion by haikeyidesu** [sst-inc/help#60] [Q&A - React] [answered]
I am not able to delete my task
![Simulator Screen Shot - iPhone 13 - 2021-11-25 at 19 10 53](https://user-images.githubusercontent.com/87898544/143432028-55219062-8aff-4e97-846d-365e90e3ce80.png)

here's my ViewTaskPage
<img width="911" alt="Screenshot 2021-11-25 at 7 14 19 PM" src="https://user-images.githubusercontent.com/87898544/143432149-42eb40dc-9787-44fb-a1b3-aa8913b0a7b3.png">

**Comment by ghost** [comment, marked as answer]
Can you send your index.js here?
